packages/es-components/cypress/plugins/index.js

```diff
@@ -15,8 +15,9 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
+
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  console.debug({ on, config });
```

Comment:
Previously, there were lint errors about the `on` and `config` parameters not being used, but any
future usage is going to need to know about them. `console.debug` mostly silently uses these parameters to clear the lint error, and facilitates keeping them around for future use.